### PR TITLE
Map population to accidents #NEAF-38

### PR DESCRIPTION
**Pull request by VladimirBaklan**

### CODE REVIEW

File: src/main/scala/com/itechart/ny_accidents/parse/PopulationParser.scala [hunk 1]
```diff
@@ -10,22 +10,34 @@ import org.apache.spark.rdd.RDD
...
 import org.slf4j.LoggerFactory
 
+import scala.util.Try
 import scala.util.control.Exception
 
 @Singleton
 class PopulationParser @Inject()(districtsService: DistrictsService,
                                  districtsStorage: DistrictsStorage) {
   private lazy val logger = LoggerFactory.getLogger(getClass)
+  private lazy val POPULATION_YEAR = 2010
```
Comment (sleshJdev) [inline src/main/scala/com/itechart/ny_accidents/parse/PopulationParser.scala:20]: Does it mean that you have population data only for 2010?

Comment (VladimirBaklan) [reply src/main/scala/com/itechart/ny_accidents/parse/PopulationParser.scala:20]: Yes, because our dataset is only for 2000 and 2010

Comment (sleshJdev) [reply src/main/scala/com/itechart/ny_accidents/parse/PopulationParser.scala:20]: Hmm, it's a kind of restriction... General speaking you don't generate stats for [2000;2009] years?

**Review by sleshJdev — COMMENTED** [review]
Are tests for StringUtilsTest still valid?

File: src/main/scala/com/itechart/ny_accidents/parse/PopulationParser.scala (hunk 1 above)
Comment (VladimirBaklan) [reply src/main/scala/com/itechart/ny_accidents/parse/PopulationParser.scala:20]: Yes, we are interested in only in a latest information.

Comment (sleshJdev) [reply src/main/scala/com/itechart/ny_accidents/parse/PopulationParser.scala:20]: Ok, I see. Thanks

**Review by sleshJdev — APPROVED** [review]

**Review by AlexKostritsa — APPROVED** [review]

**Pull request merged**